zipline/finance/commission.py

```diff
@@ -17,9 +17,84 @@
 from abc import abstractmethod
 from six import with_metaclass
 
-DEFAULT_PER_SHARE_COST = 0.0075         # 0.75 cents per share
-DEFAULT_MINIMUM_COST_PER_TRADE = 1.0    # $1 per trade
-DEFAULT_FUTURE_COST_PER_TRADE = 2.35
+DEFAULT_PER_SHARE_COST = 0.0075       # 0.75 cents per share
+DEFAULT_PER_CONTRACT_COST = 0.85      # $0.85 per future contract
+DEFAULT_PER_DOLLAR_COST = 0.0015      # 0.15 cents per dollar
+DEFAULT_MINIMUM_COST_PER_TRADE = 1.0  # $1 per trade
+FUTURE_EXCHANGE_FEES_BY_SYMBOL = {    # Default fees by root symbol
```

Comment:
@jmccorriston provided this data to me, I don't know if there is an online reference somewhere.